packages/core/upload/admin/src/pages/App/index.jsx

```diff
@@ -30,12 +28,17 @@ const Upload = () => {
     setQuery({ sort: config.sort, page: 1, pageSize: config.pageSize });
   }, [isLoading, isError, config, rawQuery, setQuery]);
 
-  useFocusWhenNavigate();
+  if (isLoading) {
+    return (
+      <>
+        <Helmet title={title} />
```

Comment:
Should there be a title too even if it's not loading?

Reply:
I think it's probably fine? It'd be good to define this behaviour throughout the app though 🤔